deploy-testnet.yml

```diff
@@ -7,26 +7,40 @@ services:
     volumes:
       - machine:/var/opt/cartesi/machine-snapshots
 
-  deployer:
-    image: cartesi/rollups-cli:0.9.1
-    depends_on:
-      machine:
-        condition: service_started
+  authority-deployer:
+    image: cartesi/rollups-hardhat:1.0.0
     command:
       [
-        "create",
-        "--rpc",
-        "${RPC_URL:?undefined RPC_URL}",
-        "--mnemonic",
-        "${MNEMONIC:?undefined MNEMONIC}",
-        "--templateHashFile",
-        "/var/opt/cartesi/machine-snapshots/latest/hash",
-        "--outputFile",
-        "/deployments/${NETWORK:?undefined NETWORK}/${DAPP_NAME:?undefined DAPP_NAME}.json",
+        "deploy",
+        "--tags",
+        "Authority",
+        "--network",
+        "${NETWORK:?undefined NETWORK}",
+        "--export",
+        "/home/node/rollups.json",
       ]
+    volumes:
+      - ./deployments/${NETWORK:?undefined NETWORK}:/home/node
```

Comment:
Why map to `/home/node` instead of `/deployments`?

Reply:
It's where I placed the export json. And it's there because I need an empty directory: I want just the export json to be available on the host after this is done (it will be used by the testnet node afterwards)